cpp-package/example/README.md

```diff
@@ -0,0 +1,112 @@
+# MXNet C++ Package Examples
+
+## Building C++ examples
+
+The examples are built while building the MXNet library and cpp-package from source . However, they can be built manually as follows
+
+From cpp-package/examples directory
+
+-  Build all examples in release mode: **make all**
+-  Build all examples in debug mode: **make debug**
+
+By default, the examples are build to be run on GPU. To build examples to run on CPU:
+
+-  Release: **make all MXNET\_USE\_CPU=1**
```

Comment:
What's with the backslashes?

Reply:
The "_" character is used to render the word in italics.